src/lib/registration/connect_helpers.rb

```diff
@@ -94,28 +94,35 @@ def self.catch_registration_errors(message_prefix: "",
       rescue SUSE::Connect::ApiError => e
         log.error "Received error: #{e.response.inspect}"
         error_msg = e.message || ""
+        # TRANSLATORS: %d is an integer
+        error_code_message = _("Error code: '%d'. ") % e.code
```

Comment:
Quotation is usually used for strings, I don't think it's needed for numbers, you can easily find what was added to the message. And I'd put that on a separate line for better readability.

```suggestion
        error_code_message = _("HTTP error code: %d\n") % e.code
```